draft-irtf-cfrg-opaque.md

```diff
@@ -796,18 +780,18 @@ Steps:
 3. Output response
 ~~~
 
-#### RecoverCredentials(pwdU, metadata, request, response)
+#### RecoverCredentials(pwdU, blind, request, response)
 
 ~~~
-RecoverCredentials(pwdU, metadata, request, response)
+RecoverCredentials(pwdU, blind, request, response)
 
 Parameters:
 - params, the MHF parameters established out of band
 - Nh, the output size of the Hash function
 
 Input:
 - pwdU, an opaque byte string containing the user's password
-- metadata, a RequestMetadata structure
+- blind, a RequestMetadata structure
```

Comment:
"a RequestMetadata structure" should be replaced with "an OPRF Scalar value"

Reply:
Missed this one -- thanks!